backend/services/BaseModelService.js

```diff
@@ -1,26 +1,30 @@
 const NodeCouchDb = require('node-couchdb');
 const config = require('../config');
-const couch = new NodeCouchDb({ ...config.db });
 
 class BaseModelService {
-    static async find(where, options = {}) {
+    constructor() {
+        this.couch = new NodeCouchDb({ ...config.db });
```

Comment:
I would like to have everything as in ERP system. So this NodeCouchDB should be factory. 

BaseModelService should be AbstractRepository. 

Some things like login/register should be in service, other methods should be in Repositories.